proc/proc_windows.go

```diff
@@ -370,12 +384,16 @@ func dwarfFromPE(f *pe.File) (*dwarf.Data, error) {
 	return dwarf.New(abbrev, nil, nil, info, line, nil, nil, str)
 }
 
-func (dbp *Process) waitForDebugEvent() (threadID, exitCode int, err error) {
+func (dbp *Process) waitForDebugEvent(blocking bool) (threadID, exitCode int, err error) {
 	var debugEvent _DEBUG_EVENT
 	shouldExit := false
 	for {
+		var milliseconds uint32 = 0
+		if blocking {
+			milliseconds = syscall.INFINITE
+		}
 		// Wait for a debug event...
-		err := _WaitForDebugEvent(&debugEvent, syscall.INFINITE)
+		err := _WaitForDebugEvent(&debugEvent, milliseconds)
```

Comment:
I wander what the err is when milliseconds is set to 0 and you have no debug events pending.